### DISCUSSION TITLE
Copilot for JetBrains prevents PyCharm from closing projects / shutting down.

### DISCUSSION THREAD

**Discussion by palle-k** [community/community#105860] [Copilot Conversations]
### Select Topic Area

Bug

### Body

With the GitHub Copilot plugin enabled in PyCharm, if I have multiple project tabs open in the same window, and I attempt to close a project that is not in the foreground, the PyCharm window becomes unresponsive. Also, if I try to quit PyCharm, the same issue occurs

**Steps to reproduce**:

1. Install GitHub Copilot in PyCharm and make sure it is activated.
2. Open two projects in PyCharm.
3. If the projects are not all in the same window, select "Window" > "Merge all Project Windows"
4. Make sure that at least one project has a run configuration. The configuration needs to be long running, e.g. a Node.js dev server, or a FastAPI app.
5. Run this configuration.
6. While the configuration is running, switch to the tab belonging to the other project
7. Hover the tab belonging to the first project, find the X button on the left and click it to close the tab. 

**Expected Results**:

The project is closed.

**Actual Results**:

The PyCharm window becomes unresponsive. No popup appears.

**Workarounds**:

1. Stop the process before closing the project.
2. The menu bar ("PyCharm", "File", "Edit", "View", "Navigate", ...) remains responsive, so navigate to the stuck project by clicking "Window" > <name of the stuck project>. The popup will appear, and the user can kill the process to close the project.

**Configuration**:
PyCharm 2023.3.3 (Professional Edition) Build #PY-233.13763.11 Runtime version: 17.0.9+7-b1087.11 aarch64
GitHub Copilot 1.4.13.4415
macOS 14.3 (23D56)

**Related Issues**:
The following issues document a similar behavior:
- https://github.com/orgs/community/discussions/88599
- https://github.com/orgs/community/discussions/88527

**Comment by montella1507** [comment]
https://youtrack.jetbrains.com/issue/IDEA-343537 

i can confirm. Check the issue above. Copilot is blocking ALL jetbrains IDEs
